### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by oriontvv**
This PR adds dependabot config

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```
Comment (vorner) [inline .github/dependabot.yml:11]: There's no docker in this repo.

**Review by vorner — COMMENTED** [review]
Here I'm a bit on the fence if it makes sense for this particular repo. There's really only a single „real“ dependency, `serde` and I don't expect it to ever go from the `1` major version. The rest is dev dependencies and downstream users don't really care about these…

But maybe, why not :shrug:

File: .github/dependabot.yml (hunk 1 above)
Comment (oriontvv) [reply .github/dependabot.yml]: Fixed. It would also check for actions and dev-dependencies as well.

**Comment by vorner** [comment]
Thank you

**Pull request merged**